monkey/infection_monkey/system_info/windows_info_collector.py

```diff
@@ -46,16 +47,33 @@ def get_info(self):
         return self.info
 
     def get_installed_packages(self):
-        LOG.info('getting installed packages')
-        self.info["installed_packages"] = os.popen("dism /online /get-packages").read()
-        self.info["installed_features"] = os.popen("dism /online /get-features").read()
+        LOG.info('Getting installed packages')
+        system32_dir = os.environ["windir"] + "\\system32"
+        go_to_system32_dir_cmd = f"cd {system32_dir}"  # dism only works in %windir%\system32 (with admin privileges)
+
+        packages = subprocess.Popen(f"{go_to_system32_dir_cmd} && dism /online /get-packages",
+                                    shell=True,
+                                    stdout=subprocess.PIPE).stdout.read()
```

Comment:
why not use [check_output](https://docs.python.org/3.7/library/subprocess.html#subprocess.check_output)?